### DISCUSSION TITLE
inspiron 3525 compatibilty?

### DISCUSSION THREAD

**Discussion by remixedcat** [alexVinarskis/dell-powermanager#43] [General]
will this and the dell command util work w my inspiron 12 3525 on mint linux 21.2?

**Comment by alexVinarskis** [comment]
Hey,
It may, or may not :) Best way to know is to try it out. Supported models depend on what dell CCTK features are supported by the platform.

If you try it, you can submit a [test result over here](https://github.com/alexVinarskis/dell-powermanager/issues/new?assignees=alexVinarskis&labels=&projects=&template=tested_device.md&title=%5BTESTED+DEVICE%5D)
